### PR TITLE
Better error reporting in deposit script

### PR DESCRIPTION
**Pull request by dapplion**

### CODE REVIEW

File: scripts/deposit.js [hunk 1]
```diff
@@ -154,4 +176,14 @@ async function getPastLogs(contract, event, { fromBlock, toBlock }) {
...
 }
 
+function readJson(filepath) {
+  const s = fs.readFileSync(filepath)
+  try {
+    return JSON.parse(s)
+  } catch (e) {
+    e.message = `error parsing JSON string: ${e.message}\n${s}` 
+    throw e
+  }
```
Comment (filoozom) [inline scripts/deposit.js:186]: I find this a bit strange personally and feel that it shouldn't add much value compared to just not catching that error, but I haven't compared them locally to check.

**Review by filoozom — APPROVED** [review]
We should probably add a `.env.chiado` file or something, but that would be in a different PR anyways.

File: scripts/deposit.js (hunk 1 above)
Comment (dapplion) [reply scripts/deposit.js:186]: I dumps the JSON on the terminal, which the original error does not.

Comment (filoozom) [reply scripts/deposit.js:186]: Oh, I missed the `${s}`, my bad 😅

**Pull request merged**